src/rust/src/http_reader.rs

```diff
@@ -58,15 +60,19 @@ impl HttpFgbReader {
         if bytes != MAGIC_BYTES {
```

Comment:
Do we need a more permissive version check here, like you did with the file reader?

Reply:
Yes, we should have the same. Forgot it.